### PR TITLE
Example: random

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

**Comment by tshemsedinov** [comment]
@vnikiforova Here is your task, please see invitation to organization kse-ua in email to have access to this repo for review

File: Tasks/random.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Generate random integer value in given range
+
+Random = (min, max) => {
+if ( max === undefined ){;
+    max = min; return Math.floor(Math.random() * (max + 1));
+    ;} else {;
```
Comment (vnikiforova) [inline Tasks/random.js:6]:
```suggestion
    } else {
```
Remove two ;

File: Tasks/random.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    return min + Math.floor(Math.random() * (max - min + 1));
+};
```
Comment (vnikiforova) [inline Tasks/random.js:8]:
```suggestion
  }
```
Add space before bracket and remove ;

**Review by vnikiforova — CHANGES_REQUESTED** [review]
Change code style:
1. Add space before square bracket and remove ;
2. Split the string
3. Remove two ;
4. Add space before bracket and remove ;

**Comment by vnikiforova** [comment]
```js
function between(min, max) {  
  return Math.floor(
    Math.random() * (max - min + 1) + min
  )
}

console.log(between(1, 9))
```

**Comment by vnikiforova** [comment]
```py
import random


def random_num(min_n, max_n):
    print(random.randint(min_n, max_n))


random_num(1, 10)
```

**Pull request closed without merging**